docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -123,6 +124,14 @@ The root `status` indicates the `status` of the system overall.
 
 The Runtime `status` indicates whether task executions have exceeded any of the <<task-manager-configuring-health-monitoring,configured health thresholds>>. An `OK` status means none of the threshold have been exceeded. A `Warning` status means that at least one warning threshold has been exceeded. An `Error` status means that at least one error threshold has been exceeded.
 
+[IMPORTANT]
+==============================================
+Some tasks (such as <<action-types,connectors>>) will incorrectly report their status as successful even if the task failed.
```

Comment:
Sure!

Reply:
What are you thinking it should say? actions are the only known culprit of this and it does say `Some tasks` to make it seem like it's not an _only action_ thing. WDYT?

Reply:
I'm just wondering if it might lead people reading these docs to try to use the event log to look up failures for other tasks.

@gchaps Any suggestions for wording?

Reply:
I need more information to provide wording.  How will users know that the status is incorrect and what should they do?

Reply:
Oh sorry, I missed this. I saw you added feedback. Did you still need more context?